common/src/main/java/com/microsoft/identity/common/internal/eststelemetry/EstsTelemetry.java

```diff
@@ -74,20 +69,26 @@ public static synchronized EstsTelemetry getInstance() {
         return sEstsTelemetryInstance;
     }
 
+    private void setupLastRequestTelemetryCache(@NonNull final Context context) {
+        this.mLastRequestTelemetryCache = createLastRequestTelemetryCache(context);
+    }
+
     /**
-     * Initializes the Ests Telemetry by creating a new instance of {@link EstsTelemetry}
+     * Initializes the Ests Telemetry Cache by creating a new instance of {@link EstsTelemetry} if
+     * an instance already didn't exist, and then creates and sets the last request telemetry cache
+     * on that instance.
      *
      * @param context the application context
      */
-    public static void initializeEstsTelemetry(@NonNull final Context context) {
-        final String methodName = ":initializeEstsTelemetry";
+    public static synchronized void initializeEstsTelemetryCache(@NonNull final Context context) {
```

Comment:
why `static`, can be an instance method right?

Reply:
This needs to be called from anywhere. I should probably rename to `initializeEstsTelemetry` as it also creates a new instance of `EstsTelemetry` if one wasn't available. But the idea is that this is the first method that calls get called before any other method from this class is called, and it will create an instance of `EstsTelemetry` and also setup the cache on that instance.

So basically this will be called in the `initializeApplication` method in `PublicClientApplication`.

We can certainly delete this method and have an instance method that takes a `context` and creates and sets the cache on that instance and actually that might be better.

Reply:
@kreedula Addressed. This method has been removed and the instance method to set the cache on the instance has been provided.